modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
Because you can't run tests in parallel, global graph...
Not only this, but we can have multiple dispatchers if we really want and "know what we are doing" :)

I am confused about "test DI outside fx", why do we care in this module about it? We take it as an interface..

Dig users should be aware of graph, otherwise it can be misused.